tests/flow/unicode/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -73,6 +73,8 @@ function utf16Length(str, pos) {
 exports[`test keys.js 1`] = `
 "({\'この事はつもり素晴らしいことさ\': \'35jL9V\'})
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-({ \"この事はつもり素晴らしいことさ\": \"35jL9V\" });
+({
+  \"この事はつもり素晴らしいことさ\": \"35jL9V\"
```

Comment:
This is a bug in the flow parser, it incorrectly computes the length of the string with unicode characters inside.